styleguide/_static/issue-templates/bug.md

```diff
@@ -0,0 +1,76 @@
+---
+name: 🐞 Bug Report 
+about: File a bug report 
+title: 🐞 <Insert Title> 
+labels: bug
+assignees: ''
+
+---
+
+### Checklist
+In order to improve the time, how long a bugfix does take, please make sure you address as much as possible of the following checklist.
+
+- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.
+- [ ] I have reproduced the issue with at least the latest released version of \<PROJECT-NAME\>.
+
+**Tip: Adding screenshots, outputs and logs help to improve clarity.**
+
+# Summary
+<!---
+Describe the bug: A clear and concise description of what the bug is.
+-->
+
+## Reproduce the Issue
+<!---
+Describe how to reproduce the bug, e.g.:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+-->
+### Steps to reproduce the behavior:
+1. <step 1> 
+2. <step 2>
+3. ....
+
+## Expected Behaviour
+<!---
+Describe the expected behaviour: A clear and concise description of what you expected to happen.
+-->
+
+## Actual Behaviour
+<!---
+Describe the actual observed behaviour: A clear and concise description of what you actually to happen.
+Screenshot: If applicable, add screenshots to help explain your problem.
+-->
+
+## Root Cause (optional)
+<!---
+If known
+-->
```

Comment:
What happened to me very often in the last time, that I opened a BUG issue, already knew the root cause, and also already knew the solution. I always asked myself if I should put the solution in the PR or already in the issue.....or in both :)

Reply:
Generally I think it definitely should be part of the issue, having it in both or reference it from within the PR is also fine imho.